tracer/security_aware_span_listener.go

```diff
@@ -0,0 +1,138 @@
+package tracer
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/thundra-io/thundra-lambda-agent-go/constants"
+	"github.com/thundra-io/thundra-lambda-agent-go/utils"
+	logger "log"
+)
+
+var defaultSecurityMessage = "Operation was blocked due to security configuration"
```

Comment:
```suggestion
var errDefaultSecurityMessage = errors.New"Operation was blocked due to security configuration")
```

it's common in go for error vars to start with an err- prefix and declared as errors in advance.